### ISSUE TITLE
Wildcard EV certificates

### ISSUE THREAD

**Issue by mimi89999** [zmap/zlint#634]
Hello,

https://cabforum.org/wp-content/uploads/CA-Browser-Forum-EV-Guidelines-1.7.8.pdf#16 states that
> Wildcard certificates are not allowed for EV Certificates except as permitted under Appendix F.

However ZLint does not detect this as an issue in EV certificates.

Example: https://crt.sh/?id=5234772527&opt=zlint
```
$ ./zlint_3.2.0_Linux_x86_64/zlint -longSummary 5234772527.crt 
| LEVEL | # OCCURRENCES |                       DETAILS                       |
+-------+---------------+-----------------------------------------------------+
|  info |             1 |                      n_subject_common_name_included |
|  warn |             1 |   w_tls_server_cert_valid_time_longer_than_397_days |
| error |             0 |                                                  -  |
| fatal |             0 |                                                  -  |
```
